src/client/src/routes/StyleguideRoute/components/ColorToClipBoard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCopy } from '@fortawesome/free-solid-svg-icons'
+import { styled } from 'rt-theme'
+
+library.add(faCopy)
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+interface ColorToClipBoardProps {
+  color: string
+  iconColor: string
+}
+export const CopyToClipBoardWrapper = styled.div`
+  visibility: hidden;
+`
+const onClick = () => {
+  alert('copied to clipboard')
```

Comment:
Is `alert` the required to be the behaviour?

Reply:
As far as I understood the requirements, I think so.